indico/htdocs/sass/partials/_toolbars.scss

```diff
@@ -156,7 +157,7 @@ $toolbar-thin-height: 1.9em;
     @extend %toolbar-group-slider;
     display: inline-block;
     padding: 0 0.4em;
-    margin-bottom: -2px;
+    margin-bottom: 0.2em;
```

Comment:
I didn't find a broken toolbar.

`.toolbar.group` is used inside `.toolbars` which have a `0.5em` margin-bottom since 4677fa05 (one day after 19cb82be). I don't know whats going on but assuming toolbars have no background and no borders¹ and you didn't optimize for toolbar being wrapped², the addition of the margin the day after cancels the negative margin.

I prefer by far removing an uncommented hack that does unknown things right now and correct an eventual glitch down the line than having someone figure out why said hack is still there and what it does months or years from now.

¹ This is the case now.
² This seems likely given your previous comment.